openghg/retrieve/_access.py

```diff
@@ -234,7 +229,9 @@ def get_obs_surface_local(
         species: Species identifier e.g. ch4 for methane.
         start_date: Output start date in a format that Pandas can interpret
         end_date: Output end date in a format that Pandas can interpret
-        inlet: Inlet height above ground level in metres
+        inlet: Inlet height above ground level in metres; use `slice(lower, upper)` to
```

Comment:
Just to make it a bit more clear that this can still just be a single value suggesting a small change to this of:
```suggestion
        inlet: Inlet height above ground level in metres; This can be a single value or `slice(lower, upper)` can be used to
```

Reply:
Okay, I'll update the docstrings this way.